src/ripple/sync/ObjectRequester.cpp

```diff
@@ -53,11 +53,11 @@ ObjectRequester::deserialize(ObjectDigest const& digest, Slice const& slice)
     else if (prefix == HashPrefix::innerNode)
     {
         assert(slice.size() == 4 + SHAMapInnerNode::branchFactor * 32);
-        if (isFullBelow(digest))
-        {
-            return;
-        }
-        auto checkpoint = requested_;
+        // There's an opportunity here to exit early if the tree rooted at
```

Comment:
When I said this had to do with online_delete, I meant the "generation" variable had to do with online_delete.

The cache needs to stay or performance will likely degrade drastically, unless you track this separately.

Reply:
Then again, it looks like you're not removing it outright--just from this new code. So it should be OK to do this.